.github/workflows/validate-binaries.yml

```diff
@@ -40,7 +44,6 @@ jobs:
     with:
       package_type: "wheel"
       version: ${{ inputs.version }}
-      os: "linux"
       channel: ${{ inputs.channel }}
       repository: "pytorch/ao"
       with_cuda: "enable"
```

Comment:
does this need to be changed or does the validate-binaries file handle enabling/disabling cuda?